openmrs-19/src/main/resources/task-data-provider.json

```diff
@@ -14,7 +14,7 @@
       ],
       "fields": [
         {
-          "displayName": "openMRS.provider.providerId",
+          "displayName": "openMRS.provider.id",
           "fieldKey": "providerId"
```

Comment:
fieldKey depends on the name of field in returning object. For example OpenMRSProvider has field "providerId" and OpenMRSPerson "id". FieldKeys are connected with Java classes variable names.  So we have to change variable names in our OpenMRS model and then we can change fieldKeys. 

The value of displayName depends on us and it is connected with messages.properties